pages/Workflow.md

```diff
@@ -10,14 +10,60 @@ A Developer wishing to design and deploy his/her own data services with Hydra, [
 has to go through a workflow that is defined by multiple steps: from defining the best Hydra/RDF/OpenAPI description of the
 resources and operations he/she is going to provide, to create the conditions for the services to be deployed by hydrus and for the agent to access them. 
 
+<img src="static/hydra workflow - horizontal.png"/>
+
 In general, the workflow is made up of these basic steps:
 1. The Developer defines an API description in the shape of an RDF-compliant or Hydra-compliant JSON-LD (Hydra is an RDF vocabulary itself) or in the shape of an OpenAPI data structure (see [OpenAPI-to-Hydra parser](https://github.com/HTTP-APIs/hydra-openapi-parser)). Any shape that is supplied it is parsed into an Hydra-compliant JSON-LD that becomes the API documentation (in Hydra parlance: the `ApiDoc`). Every REST resource is identified by an `HydraClass`.
 2. [hydrus](https://github.com/HTTP-APIs/hydrus) toolkit can parse the `ApiDoc` and define handlers to support the pusblishing of the resources as described by the Developer; if configured, it puts in place a system for credentials management to secure the endpoints with Two-factors token authentication. Every `HydraClass` has its endpoint and HTTP methods and related operations as defined by the documentation. It may be possible in future to deploy every `HydraClass` in a "database per service" fashion, all the server instances generated by the same `ApiDoc` share the same hydrus's RDF graph storage, so to allow access to other instances' resources in case of `hydra:search` (query on multiple resources) operations.
 3. [Hydra agent](https://github.com/HTTP-APIs/hydra-python-agent) reads the `ApiDoc` via the entrypoint and parses the JSON-LD into an internal representation (a graph) that allows the client to proceed to query the endpoints for its searching, storing or retrieving necessities. The internal representation may change on-the-fly or grow depending from the different `ApiDoc` that the clients reaches. The official Hydra client implementation is [Heracles.ts](https://github.com/HydraCG/Heracles.ts).
 
 In general, an Hydra network is made up by the interaction of an instance of Hydra agent with one or more instances of hydrus. An Hydra agent is an HTTP client with capabilities of translating REST queries from-to a graph query language, so to allow efficient navigation of the graph's internal representation (see [this paragraph](/Design#hydrus-as-a-full-stack-module) in Design).
 
-Follow on to [Example](Example.md) for a more in depth explanation.
+Follow [Example](Example.md) shown below for a more in depth explanation.
+
+# Flock Demo
+
+Assume a small simulation consisting of 5 different Web APIs featuring a flock of drones. 5 different APIs are-
+* Central Controller: `http://127.0.0.1:8080/api`
+* Drone 1: `http://127.0.0.1:8081/api`
+* Drone 2: `http://127.0.0.1:8081/api`
+* Drone 3: `http://127.0.0.1:8081/api`
+* Drone 4: `http://127.0.0.1:8081/api`
+
```

Comment:
All the drones have same url? Also, why are you mentioning flockdemo.hydraecosystem.org?

Reply:
No these are incorrect, I am correcting them, Correct End points of APIs used by Flock Demo are shown below - 
/api/DroneCollection/<drone_id> 
/api/DatastreamCollection/ 
/api/DroneLogCollection/ 
/api/ControllerLogCollection 
/api/HttpApiLogCollection

Reply:
No the correct endpoints for each drone use different ports (8081,8082,8083,8084). If you click on any of the drone logs in the demo you can see all this.

Reply:
Yes you are correct for drones, But I wanted to write APIs that are supported by central controller, There are 5 different APIs (as pointed by you as well). I know that drones have different APIs. I didn't intended to write them, It was written by mistake. I wanted to mention APIs supported by central controller. You will see all these corrections in my next PR. I am almost done , Will create PR in few hours

Reply:
The is just one API (the central controller) and it has several endpoints. What do you mean by 5 different APIs?

Reply:
Pardon, I mean 5 different End Points supported by the central controller.